media-sound/clementine/clementine-1.4.1~git.recipe

```diff
@@ -123,7 +125,7 @@ BUILD()
 	mkdir -p build
 	cd build
 	cmake .. $cmakeDirArgs \
-		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_BUILD_TYPE=RelDebug \
```

Comment:
Did you mean `RelWithDebInfo`?

Reply:
You can't define a debug release without a debug package, if there is not reason I would just keep release type:

```
error: invoking cmake with -DCMAKE_BUILD_TYPE=RelWithDebInfo without debug info packages specified
```

Reply:
Sorry, I didn't actually look at the full recipe whether a debuginfo is defined. In this case, keeping `Release` is probably best.

Reply:
This was meant as a comment to @kenmays who changed the build type. :)